src/ManageCourses.Domain/DatabaseAccess/ManageCoursesDbContext.cs

```diff
@@ -106,6 +106,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .HasForeignKey(ucs => new { ucs.InstCode, ucs.CrseCode })
                 .HasPrincipalKey(cc => new { cc.InstCode, cc.CrseCode });
 
+            modelBuilder.Entity<AccessRequest>()
+                .HasOne(ar => ar.Requester)
+                .WithMany(u => u.AccessRequests)
+                .HasForeignKey(ar => ar.RequesterId)
+                .OnDelete(DeleteBehavior.SetNull);
+
+            modelBuilder.Entity<UserLog>()
+                .HasOne(ul => ul.User)
+                .WithMany(u => u.UserLogs)
```

Comment:
Arguably the way it's written I think normally a McUser doesn't ever have more than 1 UserLog associated with them, but One-to-One relationships are even messier when it comes to delete behaviours :/